src/Layout.tsx

```diff
@@ -464,7 +455,7 @@ const styles =
       a:visited {
         text-decoration: none;
 
-        &[target="_blank"] {
+        &.emphasized-link {
```

Comment:
This style of link
![image](https://user-images.githubusercontent.com/1508774/141605113-db3691aa-4378-450f-a1a7-3e76a8ee2c97.png)
used to only apply to external links (`[target="_blank"]`), but now they apply to anything with the `.emphasized-link` class (because there are internal links that we want to use them for, too; just not all of them). A bit of tech debt payback.